app/initializers/extend-ember-view.js

```diff
@@ -8,7 +8,7 @@ export function initialize(/* container, application */) {
   Ember.Checkbox.reopen(SafeStyle);
 
   Ember.LinkComponent.reopen({
-    attributeBindings: ['tooltip'],
+    attributeBindings: ['tooltip', 'data-placement'],
```

Comment:
This was from @lvuch but shouldn't be needed anymore since you don't use `data-placement`